src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  margin: 20px 8px 8px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  color: #3d4e61;
+  border: solid 1px #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  margin-bottom: 20px;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order